.packit.yml

```diff
@@ -72,6 +72,9 @@ jobs:
     trigger: commit
     owner: '@centos-automotive-sig'
     project: bluechi-snapshot
+    notifications:
+      failure_comment:
+        message: "BlueChi build failed for merged commit {commit_sha}. Please check logs {logs_url}, packit dashboard {packit_dashboard_url} and external service dashboard {external_dashboard_url}"
```

Comment:
What do you think about tagging people or a group, e.g. `@eclipse-bluechi/automotive-bluechi-project-leads`?

Reply:
Do we have such group?

Reply:
Yes, there is the project leads group (that I linked) and one for committer. I think committer in general includes too many people and leads (only 3 people) not enough. Either we ask one to be created or we simply list the volunteers directly.

Reply:
Hmm, but just adding a comment to the PR GH should send an email to all CODEOWNERS (as they are assigned to PR as reviewers) and all other engineers who posted a comment on a PR.
But if you think it's worth to tag project leads group directly, I can add it to the message

Reply:
Yes, you are right. I don't receive emails for BlueChi, but other repos, interestingly. The GH notifications on the website are definitely there then. We can still improve later if necessary.